mars_api/views.py

```diff
@@ -1,9 +1,31 @@
 from rest_framework import viewsets
 
-from mars_api.models import Player
-from mars_api.serializers import PlayerSerializer
+from mars_api.models import Game, Player, PlayerScore
+from mars_api.serializers import (
+    GameAndPlayersScoresSerializer,
+    GameSerializer,
+    PlayerScoreSerializer,
+    PlayerSerializer,
+)
 
 
 class PlayerViewSet(viewsets.ModelViewSet):
     queryset = Player.objects.all()
     serializer_class = PlayerSerializer
+
+
+class GameViewSet(viewsets.ModelViewSet):
+    queryset = Game.objects.all()
+    serializer_class = GameSerializer
+
+
+class PlayerScoreViewSet(viewsets.ModelViewSet):
+    queryset = PlayerScore.objects.all()
+    serializer_class = PlayerScoreSerializer
+
+
+class GameScoresViewSet(viewsets.ModelViewSet):
+    """A viewset for Game and related PlayerScores."""
```

Comment:
If you have a docstring here then you should make the rest consistent

Reply:
I thought I will add one here because this viewset doesn't sound trivial. Others are just straight forward model viewsets

Reply:
Fair! Also, you may want to prefetch or preselect your queryset - install `django-silk` and check how many queries are getting made